### PR TITLE
azure-pipelines: fix msys2 install

### PR DESCRIPTION
**Pull request by marler8997**
Fix issue https://github.com/ziglang/zig/issues/5679

### CODE REVIEW

File: ci/azure/pipelines.yml [hunk 1]
```diff
@@ -40,12 +40,15 @@ jobs:
   timeoutInMinutes: 360
 
   steps:
+  - powershell: |
+      (New-Object Net.WebClient).DownloadFile("https://github.com/msys2/msys2-installer/releases/download/2020-05-22/msys2-base-x86_64-20200522.sfx.exe", "sfx.exe")
```
Comment (data-man) [inline ci/azure/pipelines.yml:44]: > 20200522

Why not 20200602?

Comment (marler8997) [reply ci/azure/pipelines.yml:44]: I just copied the one from here: https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml

Should be be using the latest one?

Comment (data-man) [reply ci/azure/pipelines.yml:44]: > Should be be using the latest one?

I don't know. It's just my habit of using the latest software. :)

Comment (marler8997) [reply ci/azure/pipelines.yml]: updated

**Comment by kubkon** [comment]
Huh, since when did our msvc job started running over the 6h limit? 😮

**Comment by marler8997** [comment]
Looks like it got stuck extracting the llvm/clang/lld file for some reason:
```
+ tar xf llvm+clang+lld-10.0.0-x86_64-windows-msvc-release-mt.tar.xz
```

File: ci/azure/pipelines.yml [hunk 2]
```diff
@@ -40,12 +40,20 @@ jobs:
...
   steps:
+  - powershell: |
+      (New-Object Net.WebClient).DownloadFile("https://github.com/msys2/msys2-installer/releases/download/2020-06-02/msys2-base-x86_64-20200602.sfx.exe", "sfx.exe")
+      .\sfx.exe -y -o\
+      del sfx.exe
+    displayName: Download/Extract/Install MSYS2
   - script: |
-      git clone https://github.com/msys2/msys2-ci-base.git %CD:~0,2%\msys64
-      %CD:~0,2%\msys64\usr\bin\rm -rf %CD:~0,2%\msys64\.git
-      set PATH=%CD:~0,2%\msys64\usr\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem
```
Comment (andrewrk) [inline ci/azure/pipelines.yml:46]: for the `tar` thing, could it be because the new script no longer sets PATH?

**Pull request closed without merging**